packages/components/src/internal/components/domainproperties/SampleFieldOptions.tsx

```diff
@@ -50,16 +65,43 @@ export class SampleFieldOptions extends PureComponent<SampleFieldProps, State> {
                 .filter(st => st.type.isInteger()) // Remove rowId duplicates
                 .toList();
 
-            this.setState({ loadingState: LoadingState.LOADED, sampleTypes });
+            this.setState({
+                loadingState: LoadingState.LOADED,
+                sampleTypes,
+                validateLookup: field.isNew() || !!field.lookupValidator,
+            });
         } catch (e) {
             console.error('Failed to load sample field information', e);
             this.setState({ loadingState: LoadingState.LOADED });
         }
     };
 
+    onFieldChange = (evt: any): void => {
+        this.props.onChange?.(evt.target.id, evt.target.value);
+    };
+
+    addLookupValidator = (evt: any): void => {
+        const { onMultiChange } = this.props;
+
+        if (onMultiChange) {
```

Comment:
Looks like `onMultiChange` is always supplied and is a required prop. Additionally, seems a bit odd to only be setting `validateLookup` state when `onMultiChange` is supplied (which is every case right now as far as I can tell). Consider:

```ts
addLookupValidator: React.ChangeEventHandler<HTMLInputElement> = (evt): void => {
    let newLookupValidator: PropertyValidator;
    if (evt.target.checked) {
        newLookupValidator = new PropertyValidator(LOOKUP_VALIDATOR_VALUES);
        this.setState({ validateLookup: true });
    } else {
        this.setState({ validateLookup: false });
    }

    // I'm not seeing `as IFieldChange` as needed
    this.props.onMultiChange(List<IFieldChange>([{ id: evt.target.id, value: newLookupValidator }]));
};
```

Reply:
Good point. Done.